lib/puppet/type/ca.rb

```diff
@@ -5,4 +5,20 @@
 
   instance_eval(&Certs::CERT_COMMON_PARAMS)
 
+  newparam(:other_certs) do
+    validate do |value|
+      unless value.is_a?(Array) || value.is_a?(String) || value.is_a?(Symbol)
```

Comment:
Does this mean you accept an array of arrays?

Reply:
No https://docs.puppet.com/puppet/5.0/custom_types.html#validation-and-munging